next_release.py

```diff
@@ -29,16 +29,33 @@ def add_release(release_name):
 
     with open(fpath) as f:
         lines = f.readlines()
-        title = "Version %s" % short(release_name)
+        title = "Version {}".format(short(release_name))
         if lines[3] == title + '\n':
-            print("changes.rst not modified (it already contains %s)" % title)
+            print("changes.rst not modified (it already contains {})".format(title))
             return
         this_version = changelog_index_template.format(title=title,
                                                        underline="=" * len(title),
                                                        fpath='./changes/' + fname)
         lines[3:3] = this_version.splitlines(True)
     with open(fpath, 'w') as f:
         f.writelines(lines)
+    with open(fpath, encoding='utf-8-sig') as f:
+        print('\n'.join(f.read().splitlines()[:20]))
+    if no('Does the full changelog look right?'):
+        exit(1)
+    call('git add {}'.format(fpath))
+    call('git commit -m "added release {} as in development in changes.rst"'.format(short(release_name)))
+
+
+def add_release(release_name, branch='master'):
+    update_changelog(release_name)
+    context = {'branch': branch,
+               'release_name': release_name+'-dev',
+               'repository': abspath(dirname(__file__)),
+               'build_dir': abspath(dirname(__file__)),
+               'public_release': True}
+    update_version(context)
```

Comment:
would it be hard to create a single commit for update_changelog & update_version? If you can do that in less than 10min, then that's probably worth it (cleaner history), otherwise nevermind.